devito/finite_difference.py

```diff
@@ -137,18 +137,23 @@ def first_derivative(*args, **kwargs):
     dim = kwargs.get('dim', x)
     diff = kwargs.get('diff', h)
     order = kwargs.get('order', 1)
-    side = kwargs.get('side', 1)
+    side = kwargs.get('side', 0)
```

Comment:
to avoid string comparison and to make it cleaner, we should have something like (u may be able to find better names):

``
class Position(object):

    """
    Here a brief description
    """

    def __init__(self, position):
        self._position = position

    def __eq__(self, other):
        return self._position == other._position

    def __repr__(self):
        # Can return a suitable representation here

left = Position(-1)
right = Position(1)
centered = Position(0)
``

Then you can import ``left, right, centered`` where you need them. This approach is easily scalable for when you'll need more than just -1, 0, 1